addons/stock/migrations/12.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_renames = {
+    'stock_location_path': [
```

Comment:
Why renaming this if at the end, you are copying this table?

Reply:
Yeah, a bit stupid. It's the habit. Are you just asking me to point the silliness or requiring me to revert this?

Reply:
Better to revert it :wink: